data/org.freedesktop.impl.portal.Access.xml

```diff
@@ -37,7 +37,8 @@
         @subtitle: Subtitle for the dialog
         @body: Body text, may be ""
         @options: Vardict with optional further information
-        @response: Numeric response
+        @response: Numeric response. A value of 0 indicates that the user has
+          agreed to grant permission. Any other value is treated as a refusal.
```

Comment:
(almost) all the other `impl.portal` just state `Numeric response` too - it'd be nice to update all those, and ideally to something like: "The value returned as the response argument in the #org.freedesktop.portal.Request::Response for the corresponding Request". This way you get the automatic link to where the documentation for the return values is.

Reply:
I'm not sure I understand. What is _"the #org.freedesktop.portal.Request::Response for the corresponding Request"_ in this case?

Reply:
Huh, indeed, this is one (of the few?) that doesn't have an immediately corresponding request in the client API. I think in that case something like "The value range of the response matches the values allowed for #org.freedesktop.portal.Request::Response" for this one. Still would be nice to update the others though.

Reply:
Updated.